File: ce/snippets/visualbasic/CRMV8/dynamicentityvb/vb/dumpsdkmessagetofilede1.vb
' <snippetdumpsdkmessagetofilede1>


 ' Retrieve all SDK Message Records.
 Dim sdkMessageQuery As QueryExpression = New QueryExpression With { _
     .EntityName = "sdkmessage", .ColumnSet = New ColumnSet("categoryname", "name", "isprivate", "availability", _
                                                            "customizationlevel", "expand", "template", "sdkmessageid")}
     'Criteria = new FilterExpression(),

 Dim sdkMessages As EntityCollection = _service.RetrieveMultiple(sdkMessageQuery)

' </snippetdumpsdkmessagetofilede1>

File: ce/customer-engagement/on-premises/snippets/csharp/CRMV8/dynamicentity/cs/dumpsdkmessagetofilede1.cs
// <snippetdumpsdkmessagetofilede1>


 // Retrieve all SDK Message Records.
 QueryExpression sdkMessageQuery = new QueryExpression
 {
     EntityName = "sdkmessage",
     ColumnSet = new ColumnSet("categoryname",
         "name", "isprivate", "availability", "customizationlevel",
         "expand", "template", "sdkmessageid"),
     //Criteria = new FilterExpression(),
 };

 EntityCollection sdkMessages = _service.RetrieveMultiple(sdkMessageQuery);

// </snippetdumpsdkmessagetofilede1>